spec.html

```diff
@@ -25522,6 +25521,7 @@
       <p>The Date constructor is the %Date% intrinsic object and the initial value of the `Date` property of the global object. When called as a constructor it creates and initializes a new Date object. When `Date` is called as a function rather than as a constructor, it returns a String representing the current time (UTC).</p>
       <p>The `Date` constructor is a single function whose behaviour is overloaded based upon the number and types of its arguments.</p>
       <p>The `Date` constructor is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `Date` behaviour must include a `super` call to the `Date` constructor to create and initialize the subclass instance with a [[DateValue]] internal slot.</p>
+      <p>The `length` property of the `Date` constructor function is 7.</p>
```

Comment:
The different constructor overloads make it difficult to decide where to place the length declaration:
 1. The first constructor definition (20.3.2.1 Date, 22.1.1.1 Array, 22.2.4.1 TypedArray)
 1. The constructor with the standard arguments (20.3.2.1 Date, 22.1.1.2 Array, 22.2.4.5 TypedArray)
 1. Or the last constructor (20.3.2.3 Date, 22.1.1.3 Array, 22.2.4.5 TypedArray)

And I'm also not sure whether or not it's *correct* to place the length declaration under a specific emu-clause, because it should actually apply to the whole constructor function and not just to one of the function overloads.

(IMO it's really a case of picking the least worst option. But I'm fine with whatever we decide to do.)